staging/src/k8s.io/client-go/tools/cache/listers.go

```diff
@@ -30,6 +30,8 @@ import (
 // AppendFunc is used to add a matching item to whatever list the caller is using
 type AppendFunc func(interface{})
 
+// ListAll used to list items according selector from Store.
```

Comment:
Correct but seems hard to parse.

"ListAll calls appendFn with each value retrieved from store which matches the selector"?

Reply:
Done. Thanks.